CvGameCoreDLL_Expansion2/CvMinorCivAI.cpp

```diff
@@ -2824,6 +2854,33 @@ void CvMinorCivQuest::DoStartQuest(int iStartTurn, PlayerTypes pCallingPlayer)
 		iNotificationX = pCity->plot()->getX();
 		iNotificationY = pCity->plot()->getY();
 	}
+	else if (m_eType == MINOR_CIV_QUEST_UNIT_GET_CITY_OCC)
+	{
+
+	CvCity* pCity = pMinor->GetMinorCivAI()->GetBestCityForQuest(m_eAssignedPlayer);
+
+	FAssertMsg(pCity != NULL, "MINOR CIV AI: For some reason we got NO_PLAYER when starting a quest for a major to destroy a city.");
+
+	if (!pCity)
+		return;
+
+	m_iData1 = pCity->plot()->getX();
+	m_iData2 = pCity->plot()->getY();
+	m_iData3 = pCity->getOwner();
+
+	pMinor->GetMinorCivAI()->SetTargetedCityX(m_eAssignedPlayer, pCity->getX());
+	pMinor->GetMinorCivAI()->SetTargetedCityY(m_eAssignedPlayer, pCity->getY());
+
+	const char* strTargetNameKey = pCity->getNameKey();
+
+	strMessage = Localization::Lookup("TXT_KEY_NOTIFICATION_QUEST_UNIT_GET_CITY_OCC");
+	strMessage << strTargetNameKey;
+	strSummary = Localization::Lookup("TXT_KEY_NOTIFICATION_SUMMARY_UNIT_GET_CITY_OCC");
+	strSummary << strTargetNameKey;
+
+	iNotificationX = pCity->plot()->getX();
+	iNotificationY = pCity->plot()->getY();
```

Comment:
Loose indentation here. Empty line at the beginning of the else if body should also be removed.